spec/configuration_spec.rb

```diff
@@ -36,6 +36,13 @@
     end
   end
 
+  describe '#exclude' do
+    it 'should parse the exclude list correctly' do
```

Comment:
It would. I notice we don't have any tests for [this class](https://github.com/xing/XNGPodsSynchronizer/blob/master/lib/updater/synchronize.rb), unless I'm missing something. I'll try to add some love to `synchronize.rb` next week :smile: